apps/console/src/public/deployment.config.json

```diff
@@ -604,7 +604,6 @@
                         "internal_guest_mgt_invite_list"
                     ],
                     "create": [
-                        "internal_guest_mgt_invite_validate",
```

Comment:
Let's update the framework, j2 template, local configs for cloud development, cloud pipeline and the newly introduced api-resources.xml as well 😪

Reply:
Framework updates - https://github.com/wso2/carbon-identity-framework/pull/5314
This feature is not in cloud deployment yet. Will mention the config update requirement in the relevant issue.